docs/detections/session-view.asciidoc

```diff
@@ -0,0 +1,72 @@
+
+[float]
+[[session-view-data]]
+====== Session View displays:
+* *Interactive and non-interactive processes:* Processes and services with or without a controlling tty.
+* *User information:* The Linux user that executed each session or process, and any exec user changes.
+* *Process and event telemetry:* Process information included in the Linux logical event model.
+* *Nested sessions:* Sessions started by processes descended from the entry session.
+* *Alerts:* Alerts in the context of the processes which caused them.
+
+[float]
+[[enable-session-view]]
+=== Enable session view data
+Session View uses process data collected by the {endpoint-sec} integration,
+but this data is not collected by default. To enable Session View data, go to *Manage* -> *Policies*
+and edit one or more of your {endpoint-sec} integration policies. On the *Policy settings* tab,
+scroll down to the Linux event collection section near the bottom of the page
+and switch on the *Include session data* toggle. Only data collected by {endpoint-sec} with this setting
+enabled can be viewed in Session View.  For more information about the additional
+fields collected by the {endpoint-sec} when this setting is enabled, see https://github.com/elastic/ecs/blob/main/rfcs/text/0030-linux-event-model.md[here].
+
+[float]
+[[open-session-view]]
+=== Open session view
+Session View is accessible from the Hosts, Alerts, and Timelines pages.
+In general, to open Session View find an event or session you wish to view,
+then click *Open Session View* under *Actions*. For example:
+
+* On the Alerts page, go to *Detect* -> *Alerts*, then scroll down to view the Alerts table.
+Events viewable in Session View have a rectangular **Open Session View** icon under **Actions**:
+[role="screenshot"]
+image::images/session-view-action-icon-detail.png[Detail of Open Session View icon,width=75%]
+
+* On the Hosts page, go to *Explore* -> *Hosts*, then select either the *Sessions* or the *Events* tab.
+From either of these tabs, click the *Open Session View* icon for an event or session.
+Labeled below are 1) the *Sessions* tab, and 2) the *Open Session View* icon:
+[role="screenshot"]
+image::images/session-view-hosts-page-sessions-tab-labeled.png[]
+
+[discrete]
+[[session-view-ui]]
+=== Session View UI
+When you click *Open Session View*, the following display appears. Labeled features are described below:
+
+[role="screenshot"]
+image::images/session-view-terminal-labeled.png[]
+
+1. The *Close Session* and *Full screen* buttons.
+2. The search bar. You can find and highlight search terms within the current session.
+The buttons on the right side of the search bar allow you to jump through search results.
+3. The *display settings* button. Click here to toggle timestamps and verbose mode.
+With verbose mode enabled, Session View shows all processes created in a session, including shell startup,
```

Comment:
Similar question here - verbose mode is written with a capital "v" in the UI and is one of the selections in the Display option menu. Since this sentence isn't a direct instruction to the user, maybe it's ok to keep the text as it is?